packages/block-library/src/video/edit.native.js

```diff
@@ -202,9 +202,9 @@ class VideoEdit extends React.Component {
 								</View>
 							);
 
-							const videoStyle = {
+							const videoContainerStyle = {
```

Comment:
Could we revert renamings if they are not done for a specific reason? they are making it hard to distinguish the real fix